Album-art is missing the "cross-fetch" dependency

![image](https://github.com/lacymorrow/album-art/assets/14353938/15ada9eb-7af6-49a6-acaa-d8295488cca5)

This error shows up when compiling a project that depends on the album-art NPM package.

Apparently `isomorphic-fetch` isn't recognized as a proper package/provider for the `fetch` function. Compiling without it yields me this error:

`./node_modules/album-art/index.js - Error: Module not found: Error: Can't resolve 'cross-fetch' in '<PROJECT>/node_modules/album-art'`


I'm using Angular and I can't install/run my project without installing `cross-fetch` into my project itself. The two packages are similar, though. With [cross-fetch](https://www.npmjs.com/package/cross-fetch) being more updated and having built-in TypeScript declarations.

I personally suggest `isomorphic-fetch` to just be replaced with `cross-fetch`.

--- a/index.js
+++ b/index.js
@@ -5,12 +5,12 @@
 	if ( typeof define === 'function' && define.amd ) {
 
 		// AMD
-		define( ['isomorphic-fetch'], cx )
+		define( ['cross-fetch'], cx )
 
 	} else if ( typeof exports === 'object' ) {
 
 		// Node, CommonJS-like
-		module.exports = cx( require( 'isomorphic-fetch' ) )
+		module.exports = cx( require( 'cross-fetch' ) )
 
 	} else {
 

--- a/package.json
+++ b/package.json
@@ -54,7 +54,6 @@
   },
   "dependencies": {
     "cross-fetch": "^4.0.0",
-    "isomorphic-fetch": "^3.0.0",
     "meow": "^4.0.0"
   }
 }

--- a/package.json
+++ b/package.json
@@ -53,6 +53,7 @@
     "np": "^8.0.4"
   },
   "dependencies": {
+    "cross-fetch": "^4.0.0",
     "isomorphic-fetch": "^3.0.0",
     "meow": "^4.0.0"
   }

